plugin-ci/orb.yml

```diff
@@ -30,7 +30,7 @@ commands:
     parameters:
       version:
         type: string
-        default: 1.21.0
+        default: 1.24.0
```

Comment:
@agnivade For my understanding: There is no way for a plugin that uses this orb to specify the golangci-lint version number, right?

Reply:
They can actually. This is just the default value if they choose not to override. See the .circleci/config.yml file in the mmctl repo.

Reply:
I see that a consumer of the orb can set the version for `golangci-lint` command, but how can this be one for a consumer of the `lint` job? Could you provide an example?

Reply:
Oh you know what, scratch what I said. mmctl just parameterizes the installation. Yeah, I think you are right. Although I would have to look up circleCI documentation to be totally sure.